### ISSUE TITLE
Lint sourc code

### ISSUE THREAD

**Issue by rugk** [westnordost/StreetComplete#1031]
I only looked at the [article](https://developer.android.com/studio/write/lint.html) for a short time, but do you use some linting in your code? Or does Android Studio do it all?

As far as I see there are, at least, ways to configure the linter and this does not seem to be used in StreetComplete.

So in any case, linting the code would be a good idea. Static code checks are always good!

**Comment by matkoniecz** [comment]
> Or does Android Studio do it all?

Yes it does. Googling using https://duckduckgo.com/?q=lint+android+studio gives https://developer.android.com/studio/write/lint.html as the first result 

Therefore this issue is already fixed.

**Comment by rugk** [comment]
You know I already linked to the exact site you mention?😂

However it also mentions an XML file for configuration. And I really think to use a linter effectively, you need to configure it and enable additional rules, which are not set by default.

And this does not seem to be done, because there is no such XML file here.

**Comment by matkoniecz** [comment]
> You know I already linked to the exact site you mention?

No,  failed to spot the link. But if you read it - the why you ask "Or does Android Studio do it all?", given that the article answers that question?

**Comment by ENT8R** [comment]
The article says:

> **The `lint.xml` file**
A configuration file that you can use to specify any lint checks that you want to exclude and to customize problem severity levels.

Which means that this file is only for disabling some checks which are not needed, but enabled by default...
